prisma-fmt/tests/code_actions/test_api.rs

```diff
@@ -12,6 +12,11 @@ pub(crate) fn test_scenario(scenario_name: &str) {
         std::fs::read_to_string(&path).unwrap()
     };
 
+    path.clear();
+    write!(path, "{}/{}/diagnostics.json", SCENARIOS_PATH, scenario_name).unwrap();
+    let diagnostics = std::fs::read_to_string(&path).unwrap_or_else(|_| String::new());
+    let diagnostics = serde_json::from_str(&diagnostics).unwrap_or_default();
```

Comment:
i don't understand the changes in this file. What wasn't working before?

Reply:
So the client sends us the diagnostics and we choose the diagnostic the action is fixing. This is matched now with the span, but in the future we should use error codes.